assets/js/event-providers/easy-digital-downloads.js

```diff
@@ -22,7 +22,46 @@
 
 	const body = jQuery( 'body' );
 
-	body.on( 'edd_cart_item_added', function () {
-		global._googlesitekit?.gtagEvent?.( 'add_to_cart' );
+	body.on( 'edd_cart_item_added', function ( event, details ) {
+		const { title, value } = parseCartItemHTML( details.cart_item );
+		const currency = global._googlesitekit.eddCurrency;
+
+		global._googlesitekit?.gtagEvent?.( 'add_to_cart', {
+			currency,
+			value,
+			items: [
+				{
+					item_name: title,
+					price: value,
+				},
+			],
+		} );
 	} );
 } )( global.jQuery );
+
+/**
+ * Parses the provided cart item HTML to extract product details.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} cartItemHTML The HTML string for the cart item.
+ * @return {Object} `title` and `value` keys.
+ */
+export const parseCartItemHTML = ( cartItemHTML ) => {
+	const parser = new DOMParser();
+	const doc = parser.parseFromString( cartItemHTML, 'text/html' );
+
+	const title =
+		doc.querySelector( '.edd-cart-item-title' )?.textContent.trim() || '';
+	const priceText =
+		doc.querySelector( '.edd-cart-item-price' )?.textContent.trim() || '';
+
+	const priceRegex = /([\d,.]+)/;
+	const match = priceText.match( priceRegex );
+	const value = match ? parseFloat( match[ 1 ].replace( /,/g, '' ) ) : 0;
```

Comment:
Is this removing commas from prices, eg. `100,000.00` to `100000.00`? This isn't reliable and we should possibly consider a more complete pricing string parser, because not all currencies/locales use commas for thousands.

Portuguese, for instance, inverts commas/periods. In Portuguese you'd write `100.000,00`. So at minimum you'd want to only remove commas that aren't the final delimiter (and also ensure at least three digits follow.